### ISSUE TITLE
Firefox Error: Secure Connection Failed

### ISSUE THREAD

**Issue by Oxker1** [notepad-plus-plus/notepad-plus-plus#14623]
### Description of the Issue
Notepad ++ website won't open in firefox. Attached is a screenshot of the error.

![nppp error](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/65622651/b486466d-2ed8-46bf-b5d5-a432e44339c9)

**Comment by rdipardo** [comment]
The issue is with Firefox: <https://bugzilla.mozilla.org/show_bug.cgi?id=1380420>

Try a different browser. Even [basic Chromium](https://github.com/ungoogled-software/ungoogled-chromium-windows#readme) works fine.

<img width="960" alt="ungoogled-chromium-120 0 6099 216" src="https://github.com/notepad-plus-plus/notepad-plus-plus/assets/59004801/8f3fe26b-594a-4255-8fd7-836d11bf3d85">

**Comment by chcg** [comment]
See also https://www.ssllabs.com/ssltest/analyze.html?d=notepad-plus-plus.org .
I have the issue also with Firefox 121.0.1 (64 bit) on Win10 22H2
Edge is working fine.

**Comment by bohni** [comment]
> The issue is with Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1380420

No, the issue is with the webservers config, as per comment in the linked bugreport (related to the TOs domain "beamdog.com"), see https://bugzilla.mozilla.org/show_bug.cgi?id=1380420#c3:

>The server is misconfigured. It's stapling an OCSP response for a certificate with serial number 15569252438517523351 while the certificate being used by the server has the serial number 1866525106972119737.

**Comment by LorenzBung** [comment]
Same issue, Debian 12 stable with Firefox 115.11.0esr

**Comment by kmpm** [comment]
ESET Antivirus complains about the same thing, OCSP-Stapling, regardless of firefox, chrome or edge.

https://support.eset.com/en/kb6258-website-certificate-revoked-is-displayed-when-visiting-legitimate-web-pages

**Comment by bohni** [comment]
As of 2024-07-06 08:36 UTC the website is working in firefox again

**Comment by LorenzBung** [comment]
Can confirm, I guess this issue can be closed then
